EpikCord/__init__.py

```diff
@@ -973,7 +975,8 @@ async def ready(self, data: dict):
                         guild_id, commands
                     )
         return None
-
+    async def command_error(self, error:Exception, interaction:BaseInteraction):
```

Comment:
Use the ApplicationCommandInteraction typehint.

Reply:
no spaces after the `:` symbol in the typehint

<img src="https://cdn.discordapp.com/emojis/966217451337576489.webp?size=96&quality=lossless">